src/main/java/jerma/Jerma.java

```diff
@@ -0,0 +1,53 @@
+package jerma;
+
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+
+import jerma.commands.Command;
+import jerma.utils.Parser;
+import jerma.utils.Storage;
+import jerma.utils.TaskList;
+import jerma.utils.Ui;
+
+public class Jerma {
+    private TaskList tasks;
+    private Ui ui;
+    private Boolean[] running;
```

Comment:
Should the name be modified so that it is a better representation of a boolean array?